src/integration.test.ts

```diff
@@ -58,51 +81,42 @@ describe("Integration Test", (): void => {
     });
   });
 
-  const mockedExec = async (load: boolean, command: string): Promise<void> => {
-    const stdout = "standard output from Bash command";
-    const stderr = "error output from Bash command";
-    child_process.exec.mockImplementationOnce(
-      (_command: any, _options: any, callback: any): any => {
-        callback(null, { stdout, stderr });
-      }
-    );
-
-    await (load ? docker.loadDockerImages() : docker.saveDockerImages());
-
+  const execAsserts = (command: string): void => {
     expect(core.getInput).nthCalledWith(1, "key", { required: true });
-    expect(core.info).nthCalledWith<[string]>(1, command);
```

Comment:
I suspect it isn't necessary to remove this provided the correct `command` is passed.

Reply:
This assert (specifically with n = 1) was appropriate when called after the `listCommand` and the `loadCommand` but not with the `saveCommand` (where n needs to be 3 in order for the test to pass).  I took this assert out of the `execAsserts` function and just wrote it into the code block of asserts following each command with the appropriate n.  Do you think that solution is ok?

Reply:
What I meant was that you could instead call `execAsserts('docker image list --format "{{ .Repository }}:{{ .Tag }}"')` and then assert that the third call to `core.info` is with `saveCommand` separately.

Reply:
I might be missing something, but I think that if I call `execAsserts('docker image list --format "{{ .Repository }}:{{ .Tag }}"')` that I will fail the following assert:

Reply:
Ah, I didn't consider that. Does it work if you replace `lastCalledWith` with `nthCalledWith` for some value of `n`?

Reply:
I ended up adding `n: number` as an argument to `execAsserts` and adding `expect(core.info).nthCalledWith<[string]>(n, command);` back into that function.  Then I am calling `execAsserts` with n = 1 for the `loadCommand` and `listCommand` scenarios and n = 3 for the `saveCommand` scenario.  It seems pretty simple and passes the test.  Let me know if that solution suits what you are hoping for!

Reply:
It is definitely a step forward, but I think it can be even simpler since I don't think it's necessary to pass `n`.

Reply:
I need to work on this one.  I'll do another force-push once I think I've gotten it.  In the meantime, I think/hope I addressed the rest of your feedback.